### PR TITLE
Updates to hungarian localization

### PR DESCRIPTION
**Pull request by czaurel**
These are mostly wording changes to clarify the meaning of the hungarian text. E.g. 'Építész' means 'architect' and common word for a UI 'builder' is 'szerkesztő' (editor). We have included some grammatical fixes too.

### CODE REVIEW

File: locales/locale.hu.ts [hunk 1]
```diff
@@ -5,97 +5,97 @@ export const locale: Locale = {
...
     { source: 'optional', target: 'választható' },
-    { source: 'choose', target: 'választ' },
-    { source: 'Clear value', target: 'Tiszta érték' },
+    { source: 'choose', target: 'válasszon' },
+    { source: 'Clear value', target: 'Érték törlése' }, 
     { source: 'loading...', target: 'Betöltés...' },
     { source: 'Nothing found', target: 'Nincs találat' },
     { source: 'Search...', target: 'Keresés...' },
-    { source: 'Enter at least {0} characters', target: 'Írjon be legalább {0} karaktert' },
+    { source: 'Enter at least {0} characters', target: 'Írjon be legalább {0} karaktert!' },
```
Comment (f1nality) [inline locales/locale.hu.ts:24]: Why adding exclamation mark? There is no such in original

File: locales/locale.hu.ts [hunk 2]
```diff
@@ -5,97 +5,97 @@ export const locale: Locale = {
...
+    { source: 'Sort 9 → 1', target: 'Rendezés 9 → 1' },
+    { source: 'Sort A → Z', target: 'Rendezés A → Z' },
+    { source: 'Sort Z → A', target: 'Rendezés Z → A' },
     { source: 'Refresh automatically', target: 'Automatikus frissítés' },
     { source: 'All', target: 'Összes' },
     { source: 'Selected', target: 'Kiválasztott' },
     {
       source: 'Unfortunately, no {0} matching your query found',
-      target: 'Sajnos nem található a lekérdezésének megfelelő {0}'
+      target: 'Sajnos nem található a lekérdezésének megfelelő elem.'
```
Comment (f1nality) [inline locales/locale.hu.ts]: Why removing {0}? There should be Collection name replacing this placeholder

File: locales/locale.hu.ts [hunk 3]
```diff
@@ -5,97 +5,97 @@ export const locale: Locale = {
...
     },
-    { source: 'We are fetching {0}, please, wait...', target: 'A(z) {0} lekérése folyamatban van, kérem, várjon...' },
+    { source: 'We are fetching {0}, please, wait...', target: 'A(z) adat lekérése folyamatban van, kérem, várjon...' },
```
Comment (f1nality) [inline locales/locale.hu.ts]: Why removing {0}? There should be Collection name replacing this placeholder

**Comment by f1nality** [comment]
Hi @czaurel first of all, thanks for your updates!

Please check the edits above, we need some feedback from your side

File: locales/locale.hu.ts (hunk 1 above)
Comment (czaurel) [reply locales/locale.hu.ts:24]: The reasoning was that grammatically correct imperative sentences in Hungarian should end with an exclamation mark.

File: locales/locale.hu.ts (hunk 2 above)
Comment (czaurel) [reply locales/locale.hu.ts]: According to our testing, this line produces the following Hungarian text when no records are found in a table:
`'Sajnos nem található a lekérdezésének megfelelő rekordokat.'`

'rekordokat' means 'records' (accusative) in Hungarian and the final sentence does not make sense in the language. The translated sentence is active so the accusative should be nominative and the subject should match the verb. So 'található'->'rekord' (singular) or 'találhatóak'->'rekordok' (plural).

If there should be a Collection name here, it seems that something is not working as expected.

File: locales/locale.hu.ts (hunk 3 above)
Comment (czaurel) [reply locales/locale.hu.ts]: The reasoning here is similar to the other case with the `{0}` placeholder, though we didn't find where the sentence is used on the UI so couldn't test it properly.

File: locales/locale.hu.ts (hunk 2 above)
Comment (f1nality) [reply locales/locale.hu.ts]: I was wrong, it is replaced with "records" label, not collection name. but can be replaced with some other labels in other templates. So if you need to display correct form, you need to edit { source: 'records', target: 'rekordok' } which will be inserted into {0} placeholder

File: locales/locale.hu.ts (hunk 3 above)
Comment (f1nality) [reply locales/locale.hu.ts]: Same here, this label { source: 'records', target: 'rekordok' } is actually being inserted into {0} placeholder. Btw use cases for this label are not actual anymore, but looks like there is no need to change it.

**Comment by czaurel** [comment]
I have pushed an update which re-introduces the placeholder and changes the translation of 'records' to a singular form in Hungarian. The singular/plural rules are a bit different than in English but hopefully this fits the current use cases.

**Comment by f1nality** [comment]
@czaurel thanks! looks good now, will be available in the next release

**Pull request merged**